blocks/editable/style.scss

```diff
@@ -28,7 +28,7 @@
 	}
 
 	code {
-		padding: 0 2px;
+		padding: 2px;
 		margin: 0 -2px;
```

Comment:
Just a suggestion... Could we try to remove this negative magic to normalise spacing? I know links have it, but that's only on select.